packages/client/src/components/form/DocumentUploadfield/SimpleDocumentUploader.tsx

```diff
@@ -52,6 +52,7 @@ type IFullProps = {
   error?: string
   disableDeleteInPreview?: boolean
   onComplete: (files: IAttachmentValue | {}) => void
+  toggleFileUploading?: (isUploading: boolean) => void
```

Comment:
It's probably best if this follows the `onSomething` naming convention. `onUploadingStateChanged` works for example